src/subscriptions-client.js

```diff
@@ -661,12 +661,18 @@ client.toArray = obj => {
 
 const detectInsights = () => {
     return cockpit.script("type insights-client", { err: "ignore" }).then(
-        () => { client.insightsAvailable = true },
-        () => {
-            PK.detect().then(pk_available => {
-                client.insightsAvailable = pk_available && client.insightsPackage;
+        () => { client.insightsAvailable = true })
+            .catch(async () => {
+                let pkgManAvailable = false;
+                try {
+                    await getPackageManager();
+                    pkgManAvailable = true;
+                } catch {
+                    pkgManAvailable = false;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-166-acfd47ee-20251211-144809-rhel-10-2-devel/Coverage/lcov/github-pr.diff.gcov.html)